### ISSUE TITLE
Sufia 6.x to Sufia PCDM Migration

### ISSUE THREAD

**Issue by cam156** [projecthydra/sufia#2135]
I would like to be able to migrate all of my Sufia 6.x objects to my Sufia PCDM repository.

To do this I must export the metadata of my Sufia 6 repository and import the metadata and content into the representative PCDM objects in my Sufia PCDM application. 

Done will look like when my Sufia PCDM application contains all of my Sufia 6.x Collections as Sufia PCDM Collections and my Sufia 6.x GenericFiles as Sufia PCDM GenericWorks and FileSets.

Done will also look like when the permissions of the PCDM Collections match the permissions of the Sufia 6.x Collections.

Done will also look like when the permissions of the PCDM GenericWork and FileSets match the permissions of the Sufia 6.x GenericFiles.

Done will also look like when the versions of the PCDM FileSet match the versions of the Sufia 6.x GenericFile versions.
 
![slide1](https://cloud.githubusercontent.com/assets/1599081/15782898/fa5acc3a-2979-11e6-8b26-cabfe213d2bf.png)

**Comment by mjgiarlo** [comment]
@cam156 @hackmastera Will the work for this ticket go into the 7.3.0 release? If so, any projections when this may be available? I'd like to schedule an initial 7.3.0 release soon after we get back from holiday break.

**Comment by cam156** [comment]
@mjgiarlo At this point there is not any effort happening on getting the migration work back into 7.3.

**Comment by hackmastera** [comment]
Yes I think the plan was for 7.2 to be the migration branch. I guess I figured migration stuff would be released as 7.2.1? If that makes sense.

**Comment by mjgiarlo** [comment]
@cam156 @hackmastera Agree w/ that. I've now created a 7.2.1 milestone for this work. Thank you!
